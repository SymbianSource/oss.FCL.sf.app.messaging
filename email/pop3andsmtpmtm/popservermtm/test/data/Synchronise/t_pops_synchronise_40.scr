+OK demon POP3 server ready (demon.co.uk)
+OK username accepted
+OK password accepted
** Read cancelled
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message26; Tue, 14 Aug 2001 16:10:55 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id af2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e5cedd0@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:42:48 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHAK8>; Tue, 14 Aug 2001 16:45:35 +0100
Message-ID: <t_pops_synchronise_message026@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 26
Date: Tue, 14 Aug 2001 16:37:47 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six
Twenty-six Twenty-six Twenty-six 
Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six
Twenty-six Twenty-six Twenty-six 
Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six
Twenty-six Twenty-six Twenty-six 
Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six Twenty-six
Twenty-six Twenty-six Twenty-six 
.
** Read cancelled
+OK 
