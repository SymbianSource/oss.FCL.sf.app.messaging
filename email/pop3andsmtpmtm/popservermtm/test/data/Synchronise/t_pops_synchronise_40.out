RETR 1
QUIT
